.prettierignore

```diff
@@ -4,4 +4,7 @@
 **/lwc/**/*.xml
 **/stencil-generated/**/*
 .sfdx
-dist
\ No newline at end of file
+dist
+**/*.d.ts
+LICENSE.md
+**/.storybook/**
```

Comment:
Added, the generated files, the .storybook files and the License.md to prettier ignore